modules/run-vault/run-vault

```diff
@@ -9,7 +9,7 @@ readonly SUPERVISOR_CONFIG_PATH="/etc/supervisor/conf.d/run-vault.conf"
 readonly DEFAULT_PORT=8200
 readonly DEFAULT_LOG_LEVEL="info"
 
-readonly AZURE_INSTANCE_METADATA_URL="http://169.254.169.254/metadata/instance?api-version=2017-07-01"
+readonly AZURE_INSTANCE_METADATA_URL="http://169.254.169.254/metadata/instance?api-version=2017-04-02"
```

Comment:
Let's go with `2017-08-01`

Reply:
no problem, I will check this

Reply:
also, there are other issues, I will create new PRs

Reply:
updated version